### PR TITLE
Add test cases for ImportDatabaseConfigurationUpdater.

### LINKED ISSUE #24397: Add test cases for ImportDatabaseConfigurationUpdater
Related to #19037.

Please add non-exceptional test cases for `ImportDatabaseConfigurationUpdater`.

Ref: `ImportDatabaseConfigurationUpdaterTest`

### PR DESCRIPTION
**Pull request by yx9o**
Fixes #24397.

---

Before committing this PR, I'm sure that I have checked the following options:
- [x] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [x] I have (or in comment I request) added corresponding labels for the pull request.
- [x] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: proxy/backend/core/src/main/java/org/apache/shardingsphere/proxy/backend/handler/distsql/ral/updatable/ImportDatabaseConfigurationUpdater.java [hunk 1]
```diff
@@ -33,6 +33,8 @@
  */
 public final class ImportDatabaseConfigurationUpdater implements RALUpdater<ImportDatabaseConfigurationStatement> {
     
+    private final YamlDatabaseConfigurationImportExecutor databaseConfigImportExecutor = new YamlDatabaseConfigurationImportExecutor();
```
Comment (RaigorJiang) [inline proxy/backend/core/src/main/java/org/apache/shardingsphere/proxy/backend/handler/distsql/ral/updatable/ImportDatabaseConfigurationUpdater.java:36]: No other function uses databaseConfigImportExecutor , can it be a local variable?

Comment (yx9o) [reply proxy/backend/core/src/main/java/org/apache/shardingsphere/proxy/backend/handler/distsql/ral/updatable/ImportDatabaseConfigurationUpdater.java:36]: The validateHandler in YamlDatabaseConfigurationImportExecutor needs a mock, and local variables seem to be unsolvable.

Comment (RaigorJiang) [reply proxy/backend/core/src/main/java/org/apache/shardingsphere/proxy/backend/handler/distsql/ral/updatable/ImportDatabaseConfigurationUpdater.java:36]: Well.

**Review by RaigorJiang — CHANGES_REQUESTED** [review]
Hi @yx9o , after this PR, is there no exception case?

File: proxy/backend/core/src/test/java/org/apache/shardingsphere/proxy/backend/handler/distsql/ral/updatable/ImportDatabaseConfigurationUpdaterTest.java [hunk 2]
```diff
@@ -50,101 +48,89 @@
...
-        featureMap.put(readwriteSplitting, "/conf/import/config-readwrite-splitting.yaml");
-        featureMap.put(databaseDiscovery, "/conf/import/config-database-discovery.yaml");
-        featureMap.put(encrypt, "/conf/import/config-encrypt.yaml");
-        featureMap.put(shadow, "/conf/import/config-shadow.yaml");
-        featureMap.put(mask, "/conf/import/config-mask.yaml");
-    }
+    private ImportDatabaseConfigurationUpdater importDatabaseConfigUpdater;
     
     @Test
+    @SneakyThrows(SQLException.class)
```
[the remark this answers is not in the document]
Comment (yx9o) [reply proxy/backend/core/src/test/java/org/apache/shardingsphere/proxy/backend/handler/distsql/ral/updatable/ImportDatabaseConfigurationUpdaterTest.java:54]: Ok

**Comment by yx9o** [comment]
> Hi @yx9o , after this PR, is there no exception case?

Yes, do I need to add one?

**Comment by RaigorJiang** [comment]
> Yes, do I need to add one?

Yes, please add some exception cases.

**Review by RaigorJiang — APPROVED** [review]

**Pull request merged**